app/models/miq_alert.rb

```diff
@@ -193,21 +193,31 @@ def evaluate(target, inputs = {})
     # If we are alerting, invoke the alert actions, then add a status so we can limit how often to alert
     # Otherwise, destroy this alert's statuses for our target
     invoke_actions(target, inputs) if result
-    add_status_post_evaluate(target, result, inputs[:description])
-
+    add_status_post_evaluate(target, result, inputs[:ems_event])
     result
   end
 
-  def add_status_post_evaluate(target, result, status_description)
+  def add_status_post_evaluate(target, result, event)
+    status_description, severity, url = parse_meta(event)
     status = miq_alert_statuses.find_or_initialize_by(:resource => target)
     status.result = result
     status.ems_id = target.try(:ems_id)
     status.description = status_description || description
+    status.severity = severity unless severity.blank?
+    status.url = url unless url.blank?
     status.evaluated_on = Time.now.utc
     status.save
     miq_alert_statuses << status
   end
 
+  def parse_meta(event)
```

Comment:
oh, sorry about that, inputs[:description] never worked and was a bug Introduces in #13653.
If I remember correctly, `inputs = {:ems_event => <Event>}`

This is the flow (probably good to document it for next time I forger)
- an EmsEvent is created calling EmsEventHelper.new(self).handle
- `EmsEventHelper.handle_alert_event` is called.
- `EmsEvent::Automate.policy` is called and calls `MiqEvent.raise_evm_event` in turn
- `raise_evm_event` calls `evaluate_alerts` that calls `evaluate_queue` that places an evaluate on the queue

I will add a spec